### ISSUE TITLE
run-alpipline doesn't seem to be able to handle the installapps in a zip after version 6.0.11

### ISSUE THREAD

**Issue by ronaldnl** [microsoft/navcontainerhelper#3483]
With the release of version 6.0.12 the handling of install apps and previous version changed.
This causes the run-alpipeline to fail with the folliwing error. And this is preventing us currently on running on the latest which with the coming breaking change makes it quite the urgent issue

Unable to extract and analyze appFile C:\ProgramData\BcContainerHelper\Extensions\buildagen001vrk-salespurchcond-cidev\
012353bd-8b2d-4efd-ba82-1e03d4c4a305\app.app

Input of the run-alpipeline command:
Set artifact = ///base/Current
Set pipelineName = SalesPurchCond-cidev
Set containerName = buildagen001vpk-salespurchcond-cidev
Set installApps = 'installApps.zip'
Set installTestApps = ''
Set previousApps = ''
Set appSourceCopMandatoryAffixes = '105FDW'
Set appSourceCopSupportedCountries = ''
Set appFolders = 'DITSalesPurchaseConditions'
Set testFolders = 'DITSalesPurchaseConditionsTest'
Set memoryLimit = '8G'
Set additionalCountries = ''
Set genericImageName = ''
Set vaultNameForLocal = 'BuildVariables'
Set installTestFramework = True
Set installTestLibraries = True
Set installPerformanceToolkit = False
Set enableCodeCop = True
Set enableAppSourceCop = True
Set enablePerTenantExtensionCop = False
Set enableUICop = True
Set doNotSignApps = True
Set doNotRunTests = False
Set doNotPerformUpgrade = False
Set cacheImage = True
Set CreateRuntimePackages = True
Set defaultTenantHasAllowAppDatabaseWrite = True
Downloading https://bccontainerhelper.azureedge.net/public/6.0.15.zip
BcContainerHelper version 6.0.15
BC.HelperFunctions emits usage statistics telemetry to Microsoft
Running on Windows, PowerShell 5.1.20348.2227

Host is azure vm with image MicrosoftWindowsServer:WindowsServer:2022-datacenter-core-g2
Docker Engine version is 26.0.0

image of the error
![image](https://github.com/microsoft/navcontainerhelper/assets/74051771/af4edc57-5e09-42c9-869b-62b39ea6eef4)

Question is what am I missing here, in the zip there is 2 appfiles and that is it.

**Comment by freddydk** [comment]
Could you try the current BcContainerHelper preview? (6.0.16)

**Comment by ronaldnl** [comment]
Yes, running it as we speak will update once I have the results.
